Libraries/LibLine/Editor.cpp

```diff
@@ -462,7 +508,7 @@ String Editor::get_line(const String& prompt)
                 m_refresh_needed = true;
                 continue;
             }
-            if (ch == 0xc) {                    // ^L
```

Comment:
AAAAAAAAAAAAAAA
WHY
you know what, I'm gonna move this comment somewhere else dammit

Reply:
Ha! take __that__ clang-format!